packages/Testing/PHPUnit/AbstractRectorTestCase.php

```diff
@@ -138,9 +138,12 @@ protected function provideConfigFilePath(): string
         return '';
     }
 
+    /**
+     * @return Iterator<SmartFileInfo>
+     */
     protected function yieldFilesFromDirectory(string $directory, string $suffix = '*.php.inc'): Iterator
     {
-        return StaticFixtureFinder::yieldDirectory($directory, $suffix);
+        return StaticFixtureFinder::yieldDirectoryExclusively($directory, $suffix);
```

Comment:
Now every fixture in a test case requires the `*.php.inc` suffix, or will throw an exception.
This will happen on any PHPUnit test run :+1: 

cc @matthiasnoback @samsonasik

Reply:
@TomasVotruba Thank you 👍